gnuradio-runtime/include/gnuradio/thrift_application_base.h

```diff
@@ -202,8 +202,8 @@ void thrift_application_base<TserverBase, TserverClass>::start_application()
             "thrift", "init_attempts", d_default_max_init_attempts));
 
     if (!p_impl->d_application_initialized) {
-        p_impl->d_start_thrift_thread.reset((new gr::thread::thread(
-            boost::bind(&thrift_application_base::start_thrift, d_application))));
+        p_impl->d_start_thrift_thread.reset(
```

Comment:
Slightly neater to change this to use `make_unique` now that we have C++14, to make it clearer from grep and reading that this is not manual memory management.

Reply:
oh, nice!  will do.

Reply:
it's a shared pointer, but `make_shared` is still a thing, so.